src/ducks/collectibles.ts

```diff
@@ -683,4 +724,108 @@ export const useCollectiblesStore = create<CollectiblesState>((set, get) => ({
         (collection) => collection.collectionAddress === collectionAddress,
       )
       ?.items.find((item) => item.tokenId === tokenId),
+
+  /**
+   * Hides a collectible by adding it to hidden collectibles storage and updating the store
+   *
+   * @param {Object} params - Parameters for hiding a collectible
+   * @param {string} params.publicKey - The public key of the account owning the collectible
+   * @param {string} params.network - The network of the collectible
+   * @param {string} params.contractId - The contract ID of the collectible
+   * @param {string} params.tokenId - The token ID of the collectible
+   * @returns {Promise<void>} Promise that resolves when hide completes
+   * @throws {Error} When storage operation fails
+   */
+  hideCollectible: async ({
+    publicKey,
+    network,
+    contractId,
+    tokenId,
+  }: {
+    publicKey: string;
+    network: string;
+    contractId: string;
+    tokenId: string;
+  }) => {
+    try {
+      // Add to hidden collectibles storage
+      await addHiddenCollectibleToStorage({
```

Comment:
related to my comment here: https://github.com/stellar/freighter-mobile/pull/600/files#r2586427355

If the `publicKey` or `publicKey[network]` is not found, `addHiddenCollectibleToStorage` will silently return, but your code will continue and `updatedCollections` down below will reflect something different than what is in storage

Reply:
Ah sorry, you're right - I misread what function was being called here. Disregard my comment